src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java

```diff
@@ -0,0 +1,47 @@
+package com.lftechnology.java.training.srijan.basics;
+
+
+import java.security.SecureRandom;
```

Comment:
You can also use Math.Random to generate numbers between [0,1]. You can then change the logic to generate numbers between any interval.